### PR TITLE
fix(subscriber-dummy): make crate version valid by adding the `patch` part

### PR DESCRIPTION
**Pull request by RRRadicalEdward**
Because of this, I can't use DevolutionsGateway as a git dependency. cargo gives a parsing error 
image

### CODE REVIEW

**Comment by RRRadicalEdward** [comment]
@CBenoit please have a look (I can't set the reviewing for you)

File: tools/subscriber-dummy/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "subscriber-dummy"
-version = "0.1"
+version = "0.1.0"
```
Comment (CBenoit) [inline tools/subscriber-dummy/Cargo.toml:3]:
```suggestion
version = "0.0.0"
```

**Review by CBenoit — APPROVED** [review]
Good catch, thank you! I modified the version to be 0.0.0 since there is no plan for publishing on crates.io

**Comment by CBenoit** [comment]
@RRRadicalEdward Our CI is not happy with the repository being a fork, I’ll look into this. Sorry for the delay. In the meantime, you can point the dependency to your fork to test compilation.

**Pull request merged**